resource/subfunctions/kerberos_realm.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Subfunctions
+---
+
+# Kerberos Realm
+
+Resource Administration functions for Kerberos Realm Administration
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .note }
+> _This may not be the only step to manage a **Kerberos Realm** in your environment._
+> _You may also have to refresh the **REALM** class to enact these changes._
+> _Please consult RACF documentation and manuals for an understanding of the **REALM** class._
+
+## `ResourceAdmin.add_kerberos_realm()`
+
+```python
+  def add_kerberos_realm( self, kerberos_realm_name: str, traits: dict = {} ) -> Union[dict, bytes]:
+```

```

Comment:
```python
def add_kerberos_realm(self, kerberos_realm_name: str, traits: dict = {}) -> Union[dict, bytes]:
```

Reply:
Note: remove leading spaces in front of function definition.

Reply:
Leading spaces and unnecessary trailing spaces and indents removed across all function definitions.